android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -630,6 +630,63 @@ internal class CommonKtTests {
         assertEquals(expectedProductIdentifier, receivedResponse?.get("productIdentifier"))
     }
 
+    @Test
```

Comment:
is it possible to add a test for that code that found the subscriptionOption by its id?